rainbow-me/browser-extension
# feat: adds Activity tab empty state

Figma link (if any): https://www.figma.com/file/hzfwKOddfRmeVO4KUsG9wM/branch/MBIC9y2TyLFVXCQw4LjeUQ/extensions-v2?node-id=6297%3A209036&t=KlZsg2pNCblwW7PY-0

<img width="374" alt="image" src="https://user-images.githubusercontent.com/3009331/215926130-b8ca347e-e0ed-4ff3-b8d6-1ca09b71327c.png">


## What changed (plus any additional context for devs)
- Adds null state for Activity tab


## Screen recordings / screenshots
<img width="353" alt="image" src="https://user-images.githubusercontent.com/3009331/215927424-a6b96182-e5af-41ba-b416-12b5b402693b.png">




## What to test
- Check new wallet address for empty state
- Check wallet with activity for activity to be filled


## Final checklist
